contrib/externs/chrome_extensions.js

```diff
@@ -10262,3 +10262,80 @@ chrome.downloads.DeterminingFilenameEvent = function() {};
  * https://developer.chrome.com/extensions/downloads#event-onDeterminingFilename
  */
 chrome.downloads.onDeterminingFilename;
+
+
+/**
+ * @const
+ * @see https://developer.chrome.com/extensions/declarativeContent
+ */
+chrome.declarativeContent = {};
+
+
+/**
+ * @typedef {!{
+ *   pageUrl: (!UrlFilter|undefined),
+ *   css: (!Array<string>|undefined),
+ *   isBookmarked: (boolean|undefined)
+ * }}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcher
+ */
+chrome.declarativeContent.PageStateMatcherLiteral;
+
+
+/**
+ * @constructor
+ * @param {!chrome.declarativeContent.PageStateMatcherLiteral=} literalValue
```

Comment:
Is this parameter really optional?

Reply:
Same question for similar constructors below.

Reply:
@brad4d I tested calling the constructor without parameters for `PageStateMatcher` and `RequestContentScript` and no error was thrown.

For `SetIcon` it throws (similar as passing null):
Uncaught TypeError: Cannot use 'in' operator to search for 'iconIndex' in undefined

All parameters are optional according to the documentation, but I can't say for sure if `literalValue` is optional.

Reply:
Thanks a lot for checking that out.
I'll submit this as-is.

Reply:
Actually, I will go ahead and make the arg required for `SetIcon`.
It may save some developers pain.